packages/teams-js/src/public/calendar.ts

```diff
@@ -75,6 +75,27 @@ export namespace calendar {
       }
     });
   }
+  /**
+   * Checks if the calendar capability is supported by the host
+   * @returns boolean to represent whether the calendar capability is supported
+   *
+   * @throws Error if {@linkcode app.initialize} has not successfully completed
+   */
+  export function joinMeeting(joinMeetingParams: JoinMeetingParams): Promise<void> {
+    return new Promise<void>((resolve) => {
+      ensureInitialized(runtime, FrameContexts.content);
+      if (!isSupported()) {
```

Comment:
It is probably worth understanding what this does, and how it works.
For our case, we only want to show joinMeeting is supported, but other hubs may not support joinMeeting.

Reply:
I see that you don't have an isSupported function in the new location of this code (meeting.ts) which is good. Most of our namespaces have isSupported in them, but meeting doesn't yet because it is very complicated. Your approach is correct for where the function is located now.